Syllabus/_child_assignments.Rmd

```diff
@@ -0,0 +1,32 @@
+<!-- This document contains assignment details that appear multiple places. -->
+
+## Grading
+***
+
+1. Attendance/Participation: 20% (Please let the instructors know if attendance will be difficult for you.)
+2. Homework: 3 x 15%
+3. Final "Project": 35%
+
+All assignments are due Wednesday, Jan 26, 2022 at 11:59pm EST.
+
+**Note**: only people taking the course for credit must turn in the assignments. However, we will evaluate all submitted assignments in case others would like feedback on their work.
+
+
+## Submitting Assignments
+***
+
+Submit each assignment to the designated [Drop Box on CoursePlus](https://courseplus.jhu.edu/core/index.cfm/go/db:main/coID/16733/).
+
+You should complete the following:
+
+1. [Data Quest: Introduction to Programming in R](https://app.dataquest.io/m/499/introduction-to-programming-in-r)
+2. Homework 2 Problem Set: HTML, Rmd, key (HTML), key (code), zip download
+3. Homework 3 Problem Set: HTML, Rmd, key (HTML), key (code), zip download
+4. Final Project (see below)
+
+## Final Projects 
+***
+
+This project should entail reading in a dataset of your choosing, doing some light data cleaning, performing some data summarization and visualization, and then doing some very light statistical analysis, like regression or chi-squared tests. Example projects can be found with the source code: [Rmd](ProjectExample/Final_Project_Example.Rmd), and the output [html](ProjectExample/Final_Project_Example.html) here.
```

Comment:
I like making lists like this into bullet points because I find them easier to read. You could make it a checklist since its markdown.

Reply:
good idea!